images/prow-tests/README.md

```diff
@@ -29,7 +29,7 @@ something complicated, do some rudimentary exploration in the image by running
 
 With an image you feel comfortable with deploying, create a PR to
 knative/test-infra and get approval. Once merged, pull upstream into your master
-branch and run `make push`. This will upload your image to the registry at
+branch and run `make cloud_build`. This will upload your image to the registry at
 http://gcr.io/knative-tests/test-infra/prow-tests with a date-commit_hash,
```

Comment:
<!--mdfmt-bot-->
Format markdown:
```suggestion
branch and run `make cloud_build`. This will upload your image to the registry
at http://gcr.io/knative-tests/test-infra/prow-tests with a date-commit_hash,
```